src/payment/strategies/ppsdk/step-handler/step-handler.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { createFormPoster } from '@bigcommerce/form-poster';
```

Comment:
Some of the branches within this test are near identical to those of the individual 'handlers'.

We could go full 'mockist' and just test this component only to the point that if routed to the correct mocked handler - but for now I went with the overlap.

As `failed` and `error` grow the mockist route (as taken with `continue`) may become necessary anyhow to avoid testing all sub-handlers at this root level.